api/users/serializers.py

```diff
@@ -289,3 +292,101 @@ class UserAccountDeactivateSerializer(BaseAPISerializer):
 
     class Meta:
         type_ = 'user-account-deactivate-form'
+
+
+class UserSettingsSerializer(JSONAPISerializer):
+    id = IDField(source='_id', read_only=True)
+    type = TypeField()
+    two_factor_enabled = ser.SerializerMethodField()
+    subscribe_osf_general_email = ser.SerializerMethodField()
+    subscribe_osf_help_email = ser.SerializerMethodField()
+
+    def get_two_factor_enabled(self, obj):
+        try:
+            two_factor = TwoFactorUserSettings.objects.get(owner_id=obj.id)
+            return not two_factor.deleted
+        except TwoFactorUserSettings.DoesNotExist:
+            return False
+
+    def get_subscribe_osf_general_email(self, obj):
+        return obj.osf_mailing_lists.get(MAILCHIMP_GENERAL_LIST, False)
```

Comment:
Looking at the data in a staging dump, it looks like the general email list is stored in `mailchimp_mailing_lists`:

```
In [6]:  for user in OSFUser.objects.all():
   ...:    print('OSF mailing lists: ' + str(user.osf_mailing_lists))
   ...:    print('Mailchimp mailing lists: ' + str(user.mailchimp_mailing_lists))
   ...:
OSF mailing lists: {u'Open Science Framework Help': True}
Mailchimp mailing lists: {u'Open Science Framework General - Staging': True}
OSF mailing lists: {u'Open Science Framework Help': True}
Mailchimp mailing lists: {u'Open Science Framework General - Staging': True}
OSF mailing lists: {u'Open Science Framework Help': True}
Mailchimp mailing lists: {u'Open Science Framework General - Staging': True}
OSF mailing lists: {u'Open Science Framework Help': True}
Mailchimp mailing lists: {}
```

Reply:
Will need to update the update method below as well.